identity/src/main/java/com/stripe/android/identity/IdentityActivity.kt

```diff
@@ -34,7 +36,10 @@ class IdentityActivity : AppCompatActivity(), CameraErrorListener {
             this,
```

Comment:
Since `CameraAdapter` needs `Activity` and `View` to initialize, and it needs to `bindToLifecycle`, will keep it in the `Activity` instead of moving it to `ViewModel`